### DISCUSSION TITLE
NPM Publish access

### DISCUSSION THREAD

**Discussion by mrmurphy** [rescript-labs/decco#98] [General]
Hey @ryb73! I just tried to cut a new release and the GH action reported:

```
error Couldn't publish package: "https://registry.npmjs.org/decco: This package requires that publishers enable TFA and provide an OTP to publish. For more info, visit: https://go.npm.me/2fa-guide"
```

I'm assuming you have access to the NPM package. Can you modify permissions so that the GH actions have permission to publish?

**Comment by mrmurphy** [comment]
Wait wait, I think I have access to this scope on NPM and can use a new token

**Comment by mrmurphy** [comment]
Okay, I've handled this for now by changing the package name on NPM to `@rescript-labs/decco`! I think that's a viable long-term solution as well, since that org matches the org name of this github repo.

**Reply by ryb73** [reply]
If you think that's the best solution then that works for me. I think it makes enough sense. Otherwise I don't mind updating the permissions so that we can continue publishing to the original `decco` package.

**Reply by mrmurphy** [reply]
@ryb73 if you're open to it, then let's update the original. Do you mind giving me publish access to that package on NPM? My username is the same over there, mrmurphy
